tests/org.jboss.tools.livereload.ui.bot.test/src/org/jboss/tools/livereload/ui/bot/test/LivereloadBaseTest.java

```diff
@@ -78,10 +94,36 @@ public void injectLivereload(String server, String projectName){
 		Shell s =new DefaultShell("LiveReload");
 		new YesButton().click();
 		new WaitWhile(new ShellIsAvailable(s));
-		new WaitWhile(new JobIsRunning());
-		
+		new WaitWhile(new JobIsRunning());		
 		new DefaultShell("Open").close();
 		new WaitWhile(new JobIsRunning());
 	}
+	
+	private class ModuleIsDeployed extends AbstractWaitCondition {
```

Comment:
There is ModuleIsInState condition in o.j.t.jsf.ui.test.utils. It may be worth a while to move it somewhere and use it. Your call...